rkt/image/asc.go

```diff
@@ -50,16 +50,20 @@ type remoteAscFetcher struct {
 func (f *remoteAscFetcher) Get(location string) (readSeekCloser, error) {
 	roc, err := getTmpROC(f.S, location)
 	if err != nil {
-		return nil, err
+		return NopReadSeekCloser(nil), err
 	}
-	defer func() { maybeClose(roc) }()
+	defer func() {
+		if roc != nil {
+			roc.Close()
+		}
+	}()
```

Comment:
`defer roc.Close()` is good enough, since `getTmpROC` now returns a nil-safe `readSeekCloser`.